src/display/annotation_layer.js

```diff
@@ -2662,6 +2677,9 @@ class AnnotationLayer {
       }
       elementParams.data = data;
       const element = AnnotationElementFactory.create(elementParams);
+      if (element.annotationEditorType > 0) {
+        this.#editableAnnotations.add(element);
+      }
```

Comment:
This looks strangely placed, since it ignores the `isRenderable` check just below.
Shouldn't this be moved down slightly, such that non-renderable annotations are still ignored here?